components/layout/Footer.tsx

```diff
@@ -7,7 +7,7 @@ const Footer = () => {
   return (
     <FooterWrapper>
       <ContentWrapper>
-        <div>
+        <footer>
```

Comment:
@Ibinola Instead of adding a footer tag here, inside the wrappers, I think it might be better if this whole thing was properly tagged as a Footer.

To do that you would need to change the base element type for the `FooterWrapper`, that happens in the `components > layout > styles.ts` file on line 215. 
There the `FooterWrapper` is being based based on a `div`, if you change that `div` to `footer` it will make the `FooterWrapper` render out as a Footer.

Then here on line 10 would have to go back to a regular `div`.